### PR TITLE
Add ping command

### PR DESCRIPTION
**Pull request by business-goose**

### CODE REVIEW

File: src/main/java/pw/kaboom/extras/Main.java [hunk 1]
```diff
@@ -73,6 +74,7 @@ public void onEnable() {
 		this.getCommand("spidey").setExecutor(new CommandSpidey());
 		this.getCommand("tellraw").setExecutor(new CommandTellraw());
 		this.getCommand("username").setExecutor(new CommandUsername());
+        this.getCommand("ping").setExecutor(new CommandPing());
```
Comment (mathiascode) [inline src/main/java/pw/kaboom/extras/Main.java:77]: Tabs, alpha-sort.

File: src/main/resources/plugin.yml [hunk 2]
```diff
@@ -34,6 +34,10 @@ commands:
    kaboom:
       description: I wonder...
       permission: extras.kaboom
+   ping:
+      aliases: [delay, ms]
+      description: Displays the delay of your connection to the server in milliseconds
```
Comment (mathiascode) [inline src/main/resources/plugin.yml:39]: Is this description too long for the readme?

File: src/main/java/pw/kaboom/extras/commands/CommandPing.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer;
+import org.bukkit.entity.Player;
+
+public final class CommandPing implements CommandExecutor {
+
+    public boolean onCommand(final CommandSender sender, final Command command, final String label, final String[] args) {
+        final Player player = (Player) sender;
+        final int ping = ((CraftPlayer) player).getHandle().playerConnection.player.ping;
```
Comment (mathiascode) [inline src/main/java/pw/kaboom/extras/commands/CommandPing.java:14]: Would https://papermc.io/javadocs/paper/1.16/org/bukkit/entity/Player.Spigot.html#getPing-- work here?

Comment (business-goose) [reply src/main/java/pw/kaboom/extras/commands/CommandPing.java:14]: > Would https://papermc.io/javadocs/paper/1.16/org/bukkit/entity/Player.Spigot.html#getPing-- work here?

Works fine for me. Should be much better than adding an entirely new library.

File: src/main/resources/plugin.yml (hunk 2 above)
Comment (business-goose) [reply src/main/resources/plugin.yml]: Shortened it to "Gets your ping"

**Review by mathiascode — APPROVED** [review]
Thanks!

File: src/main/java/pw/kaboom/extras/commands/CommandPing.java (hunk 3 above)
Comment (664848) [reply src/main/java/pw/kaboom/extras/commands/CommandPing.java]: nice

**Pull request merged**